languages/csharp-dotnet-core2/template/Algorithm.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>

```

Comment:
We're actually compatible with any version of netcore 2+ now! but this will be set to 2.2 as it's the latest version, today.